test/lit/passes/catch-pop-fixup.wast

```diff
@@ -1,12 +1,18 @@
 ;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
-;; RUN: wasm-opt %s --no-validation --catch-pop-fixup -all -S -o - | filecheck %s
+;; RUN: wasm-opt %s --catch-pop-fixup --no-validation -all -S -o - | filecheck %s
```

Comment:
Maybe add a comment about why no-validation is used here?

Reply:
Done